web/packages/teleterm/src/ui/TopBar/Connections/Connections.tsx

```diff
@@ -74,11 +74,27 @@ export function Connections() {
           padding (so 24px on both sides) and ConnectionsFilterableList had 300px of width.
         */}
         <Box width="324px" bg="levels.elevated">
-          <StepSlider currFlow="default" flows={stepSliderFlows} />
+          <StepSlider
+            currFlow="default"
+            flows={stepSliderFlows}
+            defaultStepIndex={stepToIndex(stepToOpen)}
+          />
         </Box>
       </Popover>
     </>
   );
 }
 
 const stepSliderFlows = { default: [ConnectionsSliderStep, VnetSliderStep] };
+
+const stepToIndex = (step: Step): number => {
+  switch (step) {
+    case 'connections':
+      return 0;
+    case 'vnet':
+      return 1;
+    default:
+      step satisfies never;
+      return 0;
```

Comment:
i never saw the `step satisfies never` before, is this equivalent to like our `assertUnreachable` which is used with enums?

Reply:
Yes, it's like `assertUnreachable`, but it's type-level only, doesn't throw a JS exception (only a compiler error) and doesn't require importing anything.